src/Compute/Compute/Generated/GalleryImageVersion/GalleryImageVersionCreateOrUpdateMethod.cs

```diff
@@ -125,6 +125,7 @@ public override void ExecuteCmdlet()
                                 Name = (string)t["Name"],
                                 RegionalReplicaCount = (int?)t["ReplicaCount"],
                                 StorageAccountType = (string)t["StorageAccountType"],
+                                ExcludeFromLatest = (bool?)t["ExcludeFromLatest"],
```

Comment:
Could you add test in these files: https://github.com/Azure/azure-powershell/tree/main/src/Compute/Compute/Generated/GalleryImageVersion

Reply:
addressed